README.md

```diff
@@ -82,6 +82,12 @@ environment.systemPackages = [
 zypper install oculante
 ```
 
+- Flatpak
+
+```sh
+flatpak install io.github.woelper.Oculante
```

Comment:
flatpak install flathub io.github.woelper.Oculante
(it will specify flathub as the remote)